Jenkinsfile

```diff
@@ -49,9 +49,9 @@ pipeline {
         stage('RSpec') {
           steps { sh 'ci/test rspec' }
         }
-        // stage('Authenticators') {
-        //   steps { sh 'ci/test cucumber_authenticators' }
-        // }
+        stage('Authenticators') {
+          steps { sh 'ci/test cucumber_authenticators' }
+        }
```

Comment:
@micahlee please note that this has changed so you'll have some conflicts. See latest jenkinsfile -https://github.com/cyberark/conjur/blob/master/Jenkinsfile#L60

Reply:
Yeah, we are aware that were going to have some heavy rebasing to do. A lot has been going on in the testing framework lately! 🙂